src/python/tests/core/test_scheduler.py

```diff
@@ -79,21 +102,22 @@ class TestScheduler:
         interval_schedule.name = 'sleep10'
         interval_schedule.process_name = "sleep10"
 
-        await scheduler.save_schedule(interval_schedule)
+        await scheduler.save_schedule(interval_schedule)  # Save update on scheduler
```

Comment:
I would avoid comments that are pretty much just method names. Sometimes these kinds of comments are useful on their own line to separate one section of the code from the other.